static/src/javascripts/bootstraps/media.js

```diff
@@ -240,7 +240,7 @@ define([
 
                     player.fullscreener();
 
-                    if (commercialFeatures.videoPreRolls && !blockVideoAds) {
+                    if (commercialFeatures.videoPreRolls && !blockVideoAds && !config.page.hasMultipleVideosInPage) {
```

Comment:
It might be worth adding a comment describing the pitfalls of running multiple prerolls on a page, and how a future developer can validate whether the problem is still an issue.